ui/components/multichain/edit-accounts-modal/edit-accounts-modal.tsx

```diff
@@ -113,6 +114,18 @@ export const EditAccountsModal: React.FC<EditAccountsModalProps> = ({
   const defaultSet = new Set(defaultSelectedAccountAddresses);
   const selectedSet = new Set(selectedAccountAddresses);
 
+  const handleNewAccount = useCallback(
+    async (completed: boolean, newAccount?: InternalAccount) => {
+      if (completed && newAccount) {
+        const { namespace, reference } = parseCaipChainId(newAccount.scopes[0]);
+        const newAccountCaipAccountId: CaipAccountId = `${namespace}:${reference}:${newAccount.address}`;
+        onSubmit([...selectedAccountAddresses, newAccountCaipAccountId]);
```

Comment:
I miss context here, but what if the accounts has multiple scopes? Given that we use it in a array, shouldn't enumerate all CAIP-10 in that list too?

Reply:
For tracking purposes: currently the multichain API does not use every CAIP-10 accounts but mostly rely on the CAIP namespace instead, thus, 1 CAIP-10 is enough.

Though, this is "sort-of" tech-debts and will be tackled later by them.